ui/css/utilities/colors.scss

```diff
@@ -12,7 +12,7 @@ Before adding a color here make sure that there isn't a design token available.
   // DEPRECATED
   // These third party network colors have been deprecated and should be removed once they are no longer in use. We should be using images to represent these networks instead.
   --color-network-goerli-default: #1098fc;
-  --color-network-sepolia-default: #cfb5f0;
+  --color-network-sepolia-default: #c65cf2;
```

Comment:
We may also want to update mobile https://vscode.dev/github/MetaMask/metamask-mobile/blob/main/app/util/networks/index.js#L101